### PR TITLE
Ellen satellite info

### PR DESCRIPTION
**Pull request by Limeiz**
Added new earths (within size limit)
User gets shown earth pic of current month
Added popup with mock data (not stylized)
Set scroll lock so user can't zoom through earth

### CODE REVIEW

File: public/index.html
Comment (mohammed-shakir) [inline public/index.html:1]: Create a separate CSS file for styling

File: public/main.js
Comment (mohammed-shakir) [inline public/main.js:48]: Add a fallback/error-handling to these two functions (in case the networking goes wrong or something else)

File: public/popup.js
Comment (mohammed-shakir) [inline public/popup.js:1]: Duplicate code: fetch('/views/popup.html')...

**Review by mohammed-shakir — COMMENTED** [review]
I am not the best js developer so i think its better if @RyuTribal  took a look into this. I added a few comments on things that need to be fixed but its nothing major, i pulled the code and tested it, it looks really nice, good job :)

But yeah i would wait until you get a second review until we merge this pr, just in case :)

**Review by sanlju — APPROVED** [review]
LGTM! I added a css directory on another branch so we can have all style in those later.

**Comment by RyuTribal** [comment]
Looks good, I just have a few concerns and questions.

- I see we now have different textures for earth at different times, I'm just not sure of the difference, because if it's a difference in shading, we could try to achieve that with the directional light we have made.
- The mock data is fine for testing, but if you would like to simulate a more realistic approach to how we are gonna fetch the data and such, I would recommend to fetch a group of id's from the back-end (or return an html list with the data, a bit hard since we do not have a templating engine yet, but you could build it up in a string at the time). After a click is done, you can fetch the rest of the data
- The way we draw satellites is fine for now, but it will quickly spiral out and drop our performance when going above a thousand or so. When it comes to drawing geometry that is exactly the same but with a few differences (position, color), your best approach is to use [Instanced rendering](https://threejs.org/docs/#api/en/objects/InstancedMesh)

**Review by RyuTribal — CHANGES_REQUESTED** [review]
Check my comment above

File: public/index.html
Comment (Limeiz) [reply public/index.html:1]: Fixed on other branch, will merge it in

**Comment by Limeiz** [comment]
> Looks good, I just have a few concerns and questions.
> 
> * I see we now have different textures for earth at different times, I'm just not sure of the difference, because if it's a difference in shading, we could try to achieve that with the directional light we have made.
> * The mock data is fine for testing, but if you would like to simulate a more realistic approach to how we are gonna fetch the data and such, I would recommend to fetch a group of id's from the back-end (or return an html list with the data, a bit hard since we do not have a templating engine yet, but you could build it up in a string at the time). After a click is done, you can fetch the rest of the data
> * The way we draw satellites is fine for now, but it will quickly spiral out and drop our performance when going above a thousand or so. When it comes to drawing geometry that is exactly the same but with a few differences (position, color), your best approach is to use [Instanced rendering](https://threejs.org/docs/#api/en/objects/InstancedMesh)

Different textures, solved

Moved the instanced rendering to this issue, so it will be addressed there instead https://github.com/SOEP-Group/SOEP-Visualizer/issues/5#issue-2528478526

Will work on the second point, fetching data from the backend and test that

**Comment by mohammed-shakir** [comment]
Left to do on this branch:
1. Fix this point that Ivan brought up:
- The mock data is fine for testing, but if you would like to simulate a more realistic approach to how we are gonna fetch the data and such, I would recommend to fetch a group of id's from the back-end (or return an html list with the data, a bit hard since we do not have a templating engine yet, but you could build it up in a string at the time). After a click is done, you can fetch the rest of the data

2. Merge with @sanlju branch called "design"

**Pull request closed without merging**
